gl_test/action.yml

```diff
@@ -41,7 +41,7 @@ runs:
     - name: Install cocotb (if required)
       if: ${{ hashFiles('test/requirements.txt') == '' }}
       shell: bash
-      run: pip install cocotb==1.8.0
+      run: pip install cocotb==1.8.0 pytest
```

Comment:
What's the rationale behind installing pytest? 

If it makes sense to install it, let's add it to the requirements.txt file in the project template (https://github.com/TinyTapeout/tt06-verilog-template/blob/main/test/requirements.txt) and pin the version.

This step is going away starting from tt07 (since we expect all projects to have a requirements.txt file with tt07)

Reply:
Its been a while since I faced the particular matter, my own commentary indicates it removes warnings in the log file output about the module not being present.  It may also be related to `coverage` testing.  But since I have run with the item present in the more complex projects I don't see the original concern/issue/warning anymore.

Reply:
Googled it - and added it to requirements.txt (you can see the explanation in the commit message):

https://github.com/TinyTapeout/tt06-verilog-template/commit/831652d7dba13b49e911d249588524281ac62add

Reply:
Sure migrating to requirements.txt will be better.  I shall do this in due course.